Composer/ScriptHandler.php

```diff
@@ -144,6 +144,22 @@ public static function clearCache(Event $event)
     }
 
     /**
+     * Warm up the Symfony cache.
```

Comment:
(1/3) "Warm" -> "Warms", for consistency with "Clears", "Installs" etc. (And while you're at it, maybe you can also fix the typo "Updated" to "Updates" for function `installRequirementsFile`)